app/models/company.rb

```diff
@@ -21,4 +21,8 @@ class Company < ApplicationRecord
   has_many :employees, -> { employee },
            source: :user, through: :users_companies_relationships
   has_many :tickets, through: :units
+
+  def responsible_users
+    units.pluck(:responsible_user_id).uniq.compact
+  end
```

Comment:
Це напевно має бути якийсь релейшин, і якщо він повертає тільки `id`  юзерів то напевно краще назвати його `responsible_user_ids`.

Reply:
Можна його ще якось так написати.
```suggestion
  def responsible_users
    User.where(id: units.select(:responsible_user_id).distinct)
  end
```